packages/remix-react/errorBoundaries.tsx

```diff
@@ -89,6 +106,14 @@ export function RemixCatchBoundary({
  */
 export function RemixRootDefaultCatchBoundary() {
   let caught = useCatch();
+  return <RemixRootDefaultCatchBoundaryImpl caught={caught} />;
+}
+
+function RemixRootDefaultCatchBoundaryImpl({
```

Comment:
Extracted internals into a component that can be used by the default Remix v2 Root Error Boundary